test/v3/routes/tracks_test.rb

```diff
@@ -28,10 +28,20 @@ def test_tracks
     Approvals.verify(blueberry, name: 'v3_track_unimplemented')
   end
 
+
   def test_readme
     get '/tracks/fake/three/readme'
 
     options = { format: :json, name: 'v3_valid_readme' }
     Approvals.verify(last_response.body, options)
   end
+
+  def test_images
+    get '/tracks/fake/docs/img/camera_test.png'
```

Comment:
It would recommend also verifying that `Content-Type` header is set to `image/png` and not `application/json`.